infra/cifuzz/platform_config/gitlab.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for getting the configuration CIFuzz needs to run on GitLab."""
+import logging
+import os
+
+import platform_config
+
+
+class PlatformConfig(platform_config.BasePlatformConfig):
+  """CI environment for GitLab."""
+
+  @property
+  def workspace(self):
+    """Returns the workspace."""
+    return os.getenv('CI_PROJECT_DIR')
+
+  @property
+  def git_sha(self):
+    """Returns the Git SHA to checkout and fuzz."""
+    return os.getenv('CI_COMMIT_SHA')
+
+  @property
+  def project_src_path(self):
+    """Returns the directory with the source of the project"""
+    return os.getenv('CI_PROJECT_DIR')
+
+  #TODO ask useful
```

Comment:
Do we need all of these ?

Reply:
See https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/platform_config/__init__.py for what's needed to override.
Off the top of my head looks like you don't strictly need to override:
```
git_sha
token
actor
project_repo_owner
git_url
```

Why did you implment all of these? is it unclear from [this comment](https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/platform_config/__init__.py#L85) that only github needs to implement project_repo_owner

Reply:
ok improved